setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="operatorcert",
+    version="1.0.0",
+    description="Tools for Red Hat Operator certification pipelines",
+    author="Red Hat, Inc.",
+    packages=find_packages(),
+    python_requires=">=3.6, <4",
+    install_requires=[
```

Comment:
Can you move it to requirements.txt and requirements-dev.txt and link files here?

Reply:
Done.